lib/src/WebSocketConnectionImpl.cc

```diff
@@ -25,6 +26,7 @@ WebSocketConnectionImpl::WebSocketConnectionImpl(
       peerAddr_(conn->peerAddr()),
       isServer_(isServer)
 {
+    conn->getLoop()->queueInLoop([this]() { setPingMessage("", 30s); });
```

Comment:
you could use `std::chrono::seconds{30}` if you don't want to use the chrono namespace